addons/account/models/account_move.py

```diff
@@ -1264,12 +1264,9 @@ def _compute_amount(self):
             total_residual_currency = 0.0
             total = 0.0
             total_currency = 0.0
-            currencies = set()
+            currencies = move._get_lines_onchange_currency().currency_id
```

Comment:
```suggestion
            currencies = move._get_lines_onchange_currency().mapped('currency_id')
```

Reply:
no... It works fine with relational field

Reply:
You are right :grimacing: